test/e2e_node/restart_test.go

```diff
@@ -117,5 +134,92 @@ var _ = framework.KubeDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
 				By(fmt.Sprintf("Docker restart test passed with %d pods", len(postRestartRunningPods)))
 			})
 		})
+		Context("Live restore", func() {
+			It("should restore container when live restore is on", func() {
+
+				// ensure live-restore is on
```

Comment:
How does this test handle older versions that do not support live restore?  Based on my reading, it looks like it would still run the restart test and make sure pods are not restarted